cabal.project

```diff
@@ -166,3 +166,7 @@ package zauth
     ghc-options: -Werror
 package fedcalls
     ghc-options: -Werror
+
+allow-newer: 
+  , proto-lens-protoc:base
+  , proto-lens-protoc:ghc
```

Comment:
Please add a comment explaining why this is necessary and why this package isn't provided by nix.